examples/step-47/doc/intro.dox

```diff
@@ -226,7 +226,7 @@ alternative method.
 We base this program on the $C^0$ IP method presented by Susanne
 Brenner and Li-Yeng Sung in the paper "C$^0$ Interior Penalty Method
```

Comment:
If we change the reference we have to change this quote - the 2010 paper is "An a posteriori error estimator for a quadratic C^0 interior penalty method for the biharmonic problem".

Reply:
Oh, I had assumed that the reason the reference at https://dealii.org/developer/doxygen/deal.II/step_47.html#DerivationoftheC0IPmethod is not properly formatted is because we meant to reference a different paper. But the issue must be with a different doxygen issue then. 

Let me try this differently then.